packages/components/src/flex/flex/hook.ts

```diff
@@ -62,7 +62,7 @@ export function useFlex( props: WordPressComponentProps< FlexProps, 'div' > ) {
 
 	const classes = useMemo( () => {
 		const base = css( {
-			alignItems: isColumn ? 'normal' : align,
+			alignItems: align,
```

Comment:
This is the most important change. I'm not sure I understand the reason for the check originally, I checked the original PR and it's not mentioned.